src/saturn/game.c

```diff
@@ -629,7 +629,9 @@ s32 Random(void) {
     return g_randomNext >> 0x18;
 }
 
-INCLUDE_ASM("asm/saturn/game/f_nonmat", f607B318, func_0607B318);
+void CreateEntityFromCurrentEntity(u16 objectId, Entity* entity);
+INCLUDE_ASM("asm/saturn/game/f_nonmat", f607B318,
```

Comment:
Oh, actually it seems like there's some build breakage due to renaming these. I'd add a comment for now, my tool doesn't do as much work as splat does.